users-api/routes/login.js

```diff
@@ -4,12 +4,19 @@ const express = require('express');
 
 const router = express.Router();
 
-router.post('/', (req,res) => {
-  if (req.body.password == 'testpass'){
-    res.send('Logged In !');
+router.put('/', (req,res) => {
+  if (req.body.password == '1234'){
+    res.status(200).send
```

Comment:
Should be just `res.json({ etc })` res.status(200) is defaulted. `.send` also sends it as plain text, which is not what we want, we would like it to respond as json.